flink-python/src/main/java/org/apache/flink/streaming/api/runners/python/beam/BeamPythonFunctionRunner.java

```diff
@@ -444,6 +450,21 @@ private ExecutableStage createExecutableStage(RunnerApi.Environment environment)
                         .putCoders(OUTPUT_CODER_ID, createCoderProto(outputCoderDescriptor))
                         .putCoders(WINDOW_CODER_ID, getWindowCoderProto());
 
+        if (sideOutputCoderDescriptors != null) {
```

Comment:
Is it a better choice to put this part of the logic in `BeamDataStreamFunctionRunner`?

Reply:
Some logic in `createExecutableStage` and `createValueOnlyWireCoderSetting` rely on this field